cmake/Modules/FindMPLIB.cmake

```diff
@@ -0,0 +1,27 @@
+find_package(GMP)
+find_package(MPIR)
+
+if(GMP_FOUND)
+    set(GR_MPLIB_GMP True)
+    set(MPLIB_DEFINITIONS "-DGR_MPLIB_GMP" ${GMP_DEFINITIONS})
+    set(MPLIB_INCLUDE_DIR ${GMP_INCLUDE_DIR})
+    set(MPLIB_LIBRARY ${GMP_LIBRARY})
+    set(MPLIBXX_LIBRARY ${GMPXX_LIBRARY})
+    set(MPLIB_PC_ADD_CFLAGS ${GMP_PC_ADD_CFLAGS})
+    set(MPLIB_PC_ADD_LIBS ${GMP_PC_ADD_LIBS})
+else(GMP_FOUND)
```

Comment:
Shouldn't we be checking whether MPIR_FOUND?

Reply:
That's a weird CMake syntax thing, not actually an "elseif(another_condition)".  See this link:
https://cmake.org/cmake/help/v3.0/command/if.html
where the "if()","else()", and "endif()" all have the same expression in parenthesis.
The expression being copied into the "else()" and endif()" is optional, according to that documentation page, so they can be removed, if you think it is too confusing.

Reply:
I get that, but I was worried that we set the variables to their MPIR values if GMP is not found, regardless of whether MPIR was found?

Reply:
Yeah I probably need to fix that.  I'm a CMake noob, so any suggestions welcome.

FWIW, hopefully the "find_package_handle_standard_args(MPLIB ..)" below won't set MPLIB_FOUND because MPIR_LIBRARY and MPIRXX_LIBRARY won't be valid/set and hence MPLIB_LIBRARY and MPLIBXX_LIBRARY won't be valid/set ... I hope. :(